### PR TITLE
Retrieve notebook token from logs

### PR DESCRIPTION
**Pull request by Yancey0623**
related issue #2366

### CODE REVIEW

**Review by lhw362950217 — APPROVED** [review]
LGTM

**Review by wangkuiyi — COMMENTED** [review]
LGTM!

File: doc/run/k8s/install-sqlflow.yaml [hunk 1]
```diff
@@ -1,59 +1,49 @@
...
-        - name: SQLFLOW_WORKFLOW_STEP_IMAGE
-          value: "sqlflow/sqlflow
+  containers:
+  - name: server
+    image: sqlflow/sqlflow
+    env:
+    - name: SQLFLOW_WORKFLOW_LOGVIEW_ENDPOINT
+      value: "http://localhost:8001"
+    - name: SQLFLOW_WORKFLOW_STEP_IMAGE
+      value: sqlflow:server
```
Comment (wangkuiyi) [inline doc/run/k8s/install-sqlflow.yaml:13]: Does this value mean the Docker image to be executed by Tekton steps?  I don't remember we have this image?

File: doc/run/k8s/install-sqlflow.yaml [hunk 2]
```diff
@@ -1,59 +1,49 @@
...
+    - containerPort: 8888
+      name: notebook
+    command:
+    - bash
+    - -c
+    - 'export SQLFLOW_DATASOURCE=mysql://root:root@tcp\(${MY_POD_IP}:3306\)/?maxAllowedPacket=0 && jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root'
+    env:
+    - name: SQLFLOW_SERVER
+      value: "localhost:50051"
+    - name: MY_POD_IP
```
Comment (wangkuiyi) [inline doc/run/k8s/install-sqlflow.yaml:29]: I think we might need to add a lot of comments in this YAML file. For example, I am not sure why do we need this environment variable `MY_POD_IP`.

File: doc/run/k8s/install-sqlflow.yaml [hunk 3]
```diff
@@ -1,59 +1,49 @@
...
+      - --argo-mode
+  - name: notebook
+    image: sqlflow/sqlflow
+    ports:
+    - containerPort: 8888
+      name: notebook
+    command:
+    - bash
+    - -c
+    - 'export SQLFLOW_DATASOURCE=mysql://root:root@tcp\(${MY_POD_IP}:3306\)/?maxAllowedPacket=0 && jupyter notebook --ip=0.0.0.0 --port=8888 --allow-root'
```
Comment (wangkuiyi) [inline doc/run/k8s/install-sqlflow.yaml:25]: This line seems that we run the Jupyter and the SQLFlow server in the same container. Should we separate the Jupyter server out as we did with the MySQL server?

File: doc/run/k8s/install-sqlflow.yaml (hunk 1 above)
Comment (Yancey0623) [reply doc/run/k8s/install-sqlflow.yaml:13]: It's used to debug, I will remove this field.

File: doc/run/k8s/install-sqlflow.yaml (hunk 3 above)
Comment (Yancey0623) [reply doc/run/k8s/install-sqlflow.yaml:25]: That's a good idea, the **all-in-one** Docker image is not needed.

**Pull request merged**